SasquatchMac/SasquatchMacSwift/AppDelegate.swift

```diff
@@ -111,6 +119,18 @@ class AppDelegate: NSObject, NSApplicationDelegate, MSCrashesDelegate, MSPushDel
     rootController.window?.makeKeyAndOrderFront(self)
   }
 
+  func overrideCountryCode() {
+    if CLLocationManager.locationServicesEnabled() {
+      self.locationManager.startUpdatingLocation()
+    }
+    else {
+      let alert : NSAlert = NSAlert()
```

Comment:
This alert isn't popped up.

Reply:
I can see the alert show up after disable the location service.
![NSAlertwindow](https://user-images.githubusercontent.com/46955844/54739386-ddfc9e80-4bf2-11e9-8c1e-81c4dbabb26d.png)